src/styles/index.scss

```diff
@@ -4,7 +4,7 @@ body {
 
 html {
   font-size: 15px;
-  margin: 0 5em;
+  margin: 0 2em;
```

Comment:
This will affect other cards on the `Tasks` page.
Can we try some other approach?